### ISSUE TITLE
Content request: best design practices for wallets (UX/UI)

### ISSUE THREAD

**Issue by konopkja** [ethereum/ethereum-org-website#13655]
This is a content request for anyone to write a beginner friendly article about designing a crypto wallet interface.

Whoever wants to write the article, please first comment on this issue and we will assign it to you.

**Where will the article live?**
It will be added to design documentation: https://ethereum.org/en/developers/docs/design-and-ux/

**Describe the solution you'd like**

- focus on practical tips how to handle different steps in the process
- describe common UX issues in wallets to fix or avoid
- guidelines to follow 

If this is too broad it can be split into multiple articles

**Comment by BoluwatifeInferno** [comment]
Gm @konopkja. I have experience in this field and would love to be assigned this task. I would love to split it into multiple articles.

**Comment by konopkja** [comment]
hi @BoluwatifeInferno  thats great! Thank you so much! Can you please create a new issue mentioning this one and explaining perhaps summary of the first article you would write?  What the sections of the article be about.

Given the topic, are you a designer?

**Comment by BoluwatifeInferno** [comment]
Yes I am, a bit rusty but I am well in tune with the basics. 

Oh sure, may I ask how to create a new issue mentioning this one?

**Comment by konopkja** [comment]
no need to create new issue - i will assign you this one.

**Comment by konopkja** [comment]
Hello @BoluwatifeInferno, do you still want to work on this one or should we give it to others in the community?

**Comment by BoluwatifeInferno** [comment]
Gm @konopkja I apologize for the late response. IRL events have had me on chokehold, but I am back and ready to finish this task. I'm sorry for the inconvenience my lack of communication caused. Still I would like to be reassigned to this task, I already have an outline and I will give myself a proper schedule to ensure I finish all the articles in this issue on time.

**Comment by wackerow** [comment]
Hey @BoluwatifeInferno! Circling back... is this something you'd still be interested in working on? Time happens, but still down to have the addition =)

**Comment by wackerow** [comment]
Opening back up if anyone would like to work on this, thanks!

**Comment by wackerow** [comment]
@BoluwatifeInferno Unassigned for now but please feel free to jump back in if you have the time 🙏

**Comment by emperor-a308** [comment]
This is much needed content! I'd suggest including sections on accessibility features and how to handle complex crypto concepts for non-technical users. Mobile-first design principles would also be valuable

**Comment by TANGAZA-CO** [comment]
Kind regards once again

**Comment by TANGAZA-CO** [comment]
I understand basic knowledge

**Comment by Daiiteamo** [comment]
#YOSOY
#REALPORQUELOSOY
#NOESPORDECIRQUE
#SOYREAL🦾🔐🎶🎒😎🎶🎶🎶

**Comment by corwintines** [comment]
@konopkja do you still see this as a priority. I would venture to say yes, just want to check.

**Comment by konopkja** [comment]
yes but because it was unsuccessful lets close this and circle back next year
